src/shared/test/java/com/couchbase/lite/ResultTest.java

```diff
@@ -1086,6 +1086,13 @@ public void testResultToMap() throws Exception {
             .where(Meta.id.equalTo(Expression.string(doc1)));
 
         ResultSet results = query.execute();
-        for (Result result : results.allResults()) { result.toMap(); }
+        assertNotNull(results);
+        for (Result result : results.allResults()) {
+            assertNotNull(result);
+            assertEquals(1, result.toMap().size());
+            Dictionary emptyDict = result.getDictionary(key1);
```

Comment:
should we change the name of the function(`testResultToMap`), since this also tests `toDictionary` and `toMap`?

Reply:
Good Idea.  Will fix in next commit.